### DISCUSSION TITLE
blog/how-to-embed-a-shiny-app-in-blogdown/

### DISCUSSION THREAD

**Discussion by giscus** [AntoineSoetewey/statsandr#101] [General]
# blog/how-to-embed-a-shiny-app-in-blogdown/

After having built your Shiny app in R, this articles shows you, step-by-step, how to embed your Shiny app into your website or blog using Hugo and blogdown

https://statsandr.com/blog/how-to-embed-a-shiny-app-in-blogdown/

**Comment by solarchemist** [comment]
Was looking to embed a Shiny app and found your post. Nice write-up, very helpful and to the point :-)

Please note that as long as you're using `<iframe>`, there is no need for `runtime: shiny` in the header. I believe that is necessary only if using the [`shinyAppDir()` function to embed](https://bookdown.org/yihui/rmarkdown/shiny-embedded.html).

**Reply by AntoineSoetewey** [reply]
Thanks for your input @solarchemist 

I have removed `runtime: shiny` in the post.
